### PR TITLE
Issues/#6879 node notifier

### PR DESCRIPTION
**Pull request by leleueri**
fixes gravitee-io/issues#6879
fixes gravitee-io/issues#6880
fixes gravitee-io/issues#6881

related to the epic gravitee-io/issues#6833

### CODE REVIEW

**Review by remisultan — COMMENTED** [review]
For Usage LGTM

A few comments.
Also the TestContainer tests fail on jdbc  NotificationAcknowledgeRepositoryTest

File: gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/notifications/EmailNotificationDefinitionBuilder.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.gravitee.am.management.service.impl.notifications;
+
+/**
+ * @author Eric LELEU (eric.leleu at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class EmailNotificationDefinitionBuilder {
```
[the remark this answers is not in the document]
Comment (leleueri) [reply gravitee-am-management-api/gravitee-am-management-api-service/src/main/java/io/gravitee/am/management/service/impl/notifications/EmailNotificationDefinitionBuilder.java:22]: it is useless indeed

**Review by remisultan — APPROVED** [review]
LGTM 👍

File: gravitee-am-model/src/main/java/io/gravitee/am/model/notification/UserNotification.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+
+import io.gravitee.am.model.ReferenceType;
+
+import java.util.Date;
+
+/**
+ * @author Eric LELEU (eric.leleu at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class UserNotification {
```
Comment (farmborough) [inline gravitee-am-model/src/main/java/io/gravitee/am/model/notification/UserNotification.java:26]: Does this pojo need to be mutable?

File: gravitee-am-model/src/main/java/io/gravitee/am/model/notification/UserNotificationContent.java [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+ */
+package io.gravitee.am.model.notification;
+
+import java.util.Date;
+
+/**
+ * @author Eric LELEU (eric.leleu at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class UserNotificationContent {
```
Comment (farmborough) [inline gravitee-am-model/src/main/java/io/gravitee/am/model/notification/UserNotificationContent.java:24]: Does this also need to be mutable?

File: gravitee-am-model/src/main/java/io/gravitee/am/model/safe/CertificateProperties.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+import java.util.Date;
+
+/**
+ * @author Eric LELEU (eric.leleu at graviteesource.com)
+ * @author GraviteeSource Team
+ */
+public class CertificateProperties {
+
+    private String id;
```
Comment (farmborough) [inline gravitee-am-model/src/main/java/io/gravitee/am/model/safe/CertificateProperties.java:28]: These could be `final`.

File: gravitee-am-model/src/main/java/io/gravitee/am/model/notification/UserNotification.java (hunk 2 above)
Comment (leleueri) [reply gravitee-am-model/src/main/java/io/gravitee/am/model/notification/UserNotification.java:26]: probably not, but I want to be consistent with all other POJO from the model package

File: gravitee-am-model/src/main/java/io/gravitee/am/model/notification/UserNotificationContent.java (hunk 3 above)
Comment (leleueri) [reply gravitee-am-model/src/main/java/io/gravitee/am/model/notification/UserNotificationContent.java:24]: probably not, but I want to be consistent with all other POJO from the model package

**Review by tcompiegne — APPROVED** [review]

**Comment by tcompiegne** [comment]
Well done 👍

**Pull request merged**